errors/errors.js

```diff
@@ -0,0 +1,64 @@
+const JSONAPIError = require('jsonapi-serializer').Error;
```

Comment:
Could you add some other errors for 403: forbidden, 409: conflict, as well as a function for returning an array of bad request errors?